sirmordred/micro-mordred.md

```diff
@@ -0,0 +1,129 @@
+
+```
+elasticsearch_1  | Search Guard Admin v6
+elasticsearch_1  | Will connect to 0.0.0.0:9300 ... done
+elasticsearch_1  | [2019-05-30T09:38:20,113][ERROR][c.f.s.a.BackendRegistry  ] Not yet initialized (you may need to run sgadmin)
+elasticsearch_1  | Elasticsearch Version: 6.1.0
+elasticsearch_1  | Search Guard Version: 6.1.0-21.0
+elasticsearch_1  | Connected as CN=kirk,OU=client,O=client,L=test,C=de
+elasticsearch_1  | Contacting elasticsearch cluster 'elasticsearch' and wait for YELLOW clusterstate ...
+elasticsearch_1  | Clustername: bitergia_elasticsearch
+elasticsearch_1  | Clusterstate: GREEN
+elasticsearch_1  | Number of nodes: 1
+elasticsearch_1  | Number of data nodes: 1
+
+...
+elasticsearch_1  | Done with success
+elasticsearch_1  | $@
+
+...
+kibiter_1        | {"type":"log","@timestamp":"2019-05-30T09:38:25Z","tags":["status","plugin:elasticsearch@6.1.4-1","info"],"pid":1,"state":"green","message":"Status changed from red to green - Ready","prevState":"red","prevMsg":"Service Unavailable"}
+```
+
+- **Note**: In case you face a memory related error, which might cause the elasticsearch instance not instantiating completely and lead the linked kibiter instance a `Request timeout`. In such a case, try adjusting the `ES_JAVA_OPTS` parameter in the *environment* attribute given in the `docker-config.yml` config file. for eg. ( -Xms1g -Xmx1g )
+
+2. At this point you should be able to access the *ElasticSearch* instance via `http://admin:admin@localhost:9200` and *Kibiter* instance via `http://admin:admin@localhost:5601` on the browser. (something like below)
+
+
+<div align="center">
+    <img scr="https://i.imgur.com/Czunlpr.png">
+    <br>
+    <p><b>Browser: Kibiter Instance</b></p>
+</div>
+
+3. We need to create and Index Pattern before we move further, but as you can see on the `Kibiter Instance` above, it say `Couldn't find any Elasticsearch data. You'll need to index some data into Elasticsearch before you can create an index pattern`. Hence, in order to do that, we'll now execute mirco-mordred using the following command, which will execute the `Raw` and `Enrich` tasks for the Git config section from the provided `setup.cfg` file.
+
+```
+$ python3 micro.py --raw --enrich setup.cfg --backends git github
+```
+
+The above command requires two files:
+  - `setup.cfg`: Contains section of configuration for different components and tools
+  - `projects.json`: Contains a list of projects to analyze
+
+Read more about the projects file [here](https://github.com/chaoss/grimoirelab-tutorial/blob/master/sirmordred/projects.md)
+
+We'll (for the purpose of this tutorial) use the files provided in the `/utils` directory, but feel free to play around with the file and their configurations :)
+
+- **Note**: In case the process fails to index the data to the ElasticSearch, check the `.perceval` folder in the home directory; which in this case may contain same repositories as mentioned in the `projects.json` file.
```

Comment:
It could be useful to add alos the action to take: `rm -rf .perceval/repositories/...`

Reply:
Yes. That would be helpful.
Thanks!